arviz/plots/compareplot.py

```diff
@@ -53,26 +53,26 @@ def plot_compare(
         If True (default) ensure the best model is used as reference.
     legend : bool
         Add legend to figure. By default True.
-    figsize : tuple, optional
+    figsize : (float, float), optional
         If None, size is (6, num of models) inches
-    title : bool:
+    title : bool
         Show a tittle with a description of how to interpret the plot. Defaults to True.
-    textsize: float
+    textsize : float
         Text size scaling factor for labels, titles and lines. If None it will be autoscaled based
         on ``figsize``.
-    labeller : labeller instance, optional
+    labeller : Labeller instance, optional

```

Comment:
```suggestion
    labeller : Labeller, optional
```

here all lowercase and uppercase is correct, but there is still the extra instance from the original docs that should be removed.